SR2023/2023-03-15-competition-details..md

```diff
@@ -0,0 +1,56 @@
+---
+to: Student Robotics 2023
+subject: SR2023 Competition Details
+---
+
+Hello all!
+
+It's only 2 weeks until the big day - [the competition](https://studentrobotics.org/events/sr2023/competition/)!
+
+Before then, we have a few important updates and reminders for you:
+
+## Media Consent
+
+Throughout the competition, Student Robotics volunteers will be taking photos and videos for the purpose of prompting and marketing the charity. The arena will also be [livestreamed](https://studentrobotics.org/events/sr2023/competition/#livestream) on our YouTube channel. If this is a problem for you or your competitors, please let our reception desk know when you arrive.
+
+The competition this year is a public event, and we do not have exclusive access to the venue.
+
+## Equipment
+
+Please ensure that your team bring at least 1 laptop or appropriate device with them to the competition to access the docs and IDE on. We are unable to provide any computing devices for competitor use at the competition. We will be providing a wireless internet connection to all teams.
```

Comment:
The framing here makes it sound like we're talking about a hosted IDE, which isn't the case. We probably want to say something like "to develop your robot code on" instead.